### PR TITLE
Refuel Jerry Can

### PR DESCRIPTION
**Pull request by tkcjesse**
Allows you to refuel your empty Jerry Can at the pump.

Tested it, works fine. Need to be within 3.5 meters and click on the use item in the player menu ( Y menu)

For issue: https://github.com/ArmaLife/Framework/issues/475

### CODE REVIEW

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -6923,6 +6923,39 @@
             <Portuguese>Você precisa estar mais perto do veículo!</Portuguese>
             <Polish>Musisz być bliżej pojazdu!</Polish>
         </Key>
+        <Key ID="STR_ISTR_Jerry_Refuel">
```
Comment (tkcjesse) [inline Altis_Life.Altis/stringtable.xml]: @Jawshy not sure if you wanna do some google translate or reach to out to whoever has been helping with certain langs.

**Comment by Jawshy** [comment]
Refuelling a fuel can does not cost anything. Was this intentional or not?

File: Altis_Life.Altis/stringtable.xml (hunk 1 above)
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: See https://github.com/ArmaLife/Framework/pull/763

**Comment by tkcjesse** [comment]
Yeah it was intentional that i never thought of making it cost something. I figured you already bought the can :p lol

**Comment by Jawshy** [comment]
Isn't that like saying you bought a car with fuel so you shouldn't have to pay to refuel it?

**Comment by tkcjesse** [comment]
No has to do with why refuel the can if you have to pay for it.. why not just click remove then buy a new can :p.

**Comment by tkcjesse** [comment]
https://github.com/ArmaLife/Framework/pull/770

**Pull request merged**